code/pizzabot.js

```diff
@@ -1,7 +1,62 @@
-const vegetarian = "Vegetarian Pizza"
-const hawaiian = "Hawaiian Pizza"
-const pepperoni = "Pepperoni Pizza"
-
-const pizzaPrice = 80
+const vegetarian = "Vegetarian Pizza";
+const hawaiian = "Hawaiian Pizza";
+const pepperoni = "Pepperoni Pizza";
+const pizzaPrice = 80;
 
 //Put your Javscript code here:
+
+const orderPizza = () => {
+  alert(
+    `Hey! Happy to serve your pizza. On our menu we have ${vegetarian}, ${hawaiian} and ${pepperoni}.`
+  );
+  const orderName = prompt(
+    "Enter the name of the pizza you want to order today."
+  );
+
+  const validateOrderName = () => {
+    if (
+      orderName === vegetarian ||
+      orderName === hawaiian ||
+      orderName === pepperoni
+    ) {
+      //checks if orderName is true, if orderName false, go to the last else
+      return true;
+    } else {
+      alert("Choose a pizza that's on the menu");
+
+      return false;
+    }
+    /*if orderName false, should prompt again, or loop from start,
+     */
+  };
+
+  if (validateOrderName(orderName)) {
+    const orderQuantity = prompt(`How many ${orderName}s do you want?`);
+    //then prompts how many pizzas you want
+    const orderTotal = orderQuantity * pizzaPrice;
```

Comment:
This line of code should be stored in separate function and called in specific moment. You should fix that